nix/default.nix

```diff
@@ -7,7 +7,7 @@ let
       # All wire-server specific packages
       (import ./overlay.nix)
       (import ./overlay-docs.nix)
-      (self: super: { lib = super.lib // (import sources.bombon).lib.${super.system}; })
+      # (self: super: { lib = super.lib // (import sources.bombon).lib.${super.system}; })
```

Comment:
Is this commented out? Do we still need it?

Reply:
Nope it should be removed. Good catch.